src/crimson/os/seastore/cached_extent.h

```diff
@@ -311,6 +426,10 @@ class CachedExtent : public boost::intrusive_ref_counter<
            state == extent_state_t::EXIST_CLEAN;
   }
 
+  bool is_clean_pending() const {
+    return state == extent_state_t::CLEAN_PENDING;
+  }
```

Comment:
I don't see who is (will be) publically caring about wether the extent is CLEAN or CLEAN_PENGING outside Cache.

Drop if unused.